utils/build.ps1

```diff
@@ -1820,13 +1917,16 @@ function Test-PackageManager() {
 }
 
 function Build-Installer($Arch) {
+  $INCLUDE_SWIFT_INSPECT = if ($IsCrossCompiling) { "false" } else { "true" }
+  $INCLUDE_SWIFT_DOCC = if ($IsCrossCompiling) { "false" } else { "true" }
```

Comment:
Might be useful to include a reference to an issue to fix this - we cannot cross-compile these just yet.

Reply:
Do you mean filing a new apple/swift issue for this and reference it?

Reply:
Yes

Reply:
Filed a issue for this https://github.com/apple/swift/issues/71655

Reply:
Added a reference. Done